### PR TITLE
#3854 Refresh KYC status

### PR DESCRIPTION
**Pull request by desfero**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -15971,6 +15971,16 @@ redux@^4.0.4:
     loose-envify "^1.4.0"
     symbol-observable "^1.2.0"
 
+redux@~3.5.2:
```
Comment (desfero) [inline yarn.lock]: seems this one was missing as `onfido-sdk-ui` depends on it. Strange that `yarn install --frozen-lockfile` didn't fail on CI

File: packages/web/webpack/webpack.config.dev.js [hunk 2]
```diff
@@ -45,7 +45,7 @@ module.exports = merge.smart(configCommon, {
         "*; " + // this should be only enabled for twitter-iframe.html
         "font-src 'self' fonts.gstatic.com; " +
         "media-src 'self' blob:; " +
-        "img-src 'self' blob: data: documents.neufund.io documents.neufund.net www.google-analytics.com stats.g.doubleclick.net " +
+        "img-src 'self' blob: data: documents.neufund.io documents.neufund.net www.google-analytics.com stats.g.doubleclick.net https://lipis.github.io/flag-icon-css/flags " +
```
Comment (desfero) [inline packages/web/webpack/webpack.config.dev.js]: https://lipis.github.io/flag-icon-css/flags is used by `onfido-sdk-ui`

File: packages/web/app/modules/kyc/sagas.ts [hunk 3]
```diff
@@ -35,18 +36,33 @@ import {
...
     const kycStatus: TKycStatus = yield apiKycService.getKycStatus();
 
     yield put(actions.kyc.setStatus(kycStatus));
+  } catch (e) {
+    logger.error("Error while getting KYC status", e);
+
+    yield put(actions.kyc.setStatusError(e.message));
+
+    // let the caller know that status is unknown
+    throw e;
```
Comment (Moejoe90) [inline packages/web/app/modules/kyc/sagas.ts]: the only issue i see here is if an error occurs here we will have two logged errors in sentry

File: packages/web/app/modules/kyc/sagas.ts [hunk 4]
```diff
@@ -146,6 +175,9 @@ function* submitPersonalDataSaga(
 ): Generator<any, any, any> {
   const result: IHttpResponse<IKycIndividualData> = yield apiKycService.putPersonalData(data);
 
+  // update kyc status after submitting personal data as it may affect supported instant id providers
+  yield neuCall(loadKycStatus);
```
Comment (Moejoe90) [inline packages/web/app/modules/kyc/sagas.ts]: and for example here do we rely on the root saga also?

File: packages/web/webpack/webpack.config.dev.js (hunk 2 above)
Comment (Moejoe90) [reply packages/web/webpack/webpack.config.dev.js]: we probably need a devops ticket for this no?

Comment (pstroinski) [reply packages/web/webpack/webpack.config.dev.js]: This is already implemented

File: packages/web/app/modules/kyc/sagas.ts (hunk 3 above)
Comment (desfero) [reply packages/web/app/modules/kyc/sagas.ts]: let's keep two logs for now

**Pull request merged**